app/controllers/reminders.js

```diff
@@ -0,0 +1,14 @@
+import Ember from 'ember';
+
+export default Ember.Controller.extend({
+  actions: {
+    remove(id) {
+      this.get('store').findRecord('reminder', id, {
+        backgroundReload: false }).then(function(post) {
```

Comment:
But also if you're going to use ES6 let's use ES6 everywhere (you have an arrow function in the next statement)